### PR TITLE
Adding Uninstall script for issue 168

### PR DESCRIPTION
**Pull request by tosin2013**
Adding clean up script for issue 168
https://github.com/redhat-et/microshift/issues/168

### CODE REVIEW

**Review by guymguym — COMMENTED** [review]
Looks good to me!

**Comment by rootfs** [comment]
thank you @guymguym 
wdyt @oglok

**Review by oglok — CHANGES_REQUESTED** [review]
Thanks @tosin2013 for pushing this PR. We really appreciate your contributions.

File: hack/cleanup.sh [hunk 1]
```diff
@@ -1,7 +1,117 @@
...
+function remove_all_pods(){
+  if [[ ! -z $(${SUDO} crictl pods | grep -v "POD ID") ]];
+  then 
+    PODS=$(${SUDO} crictl pods | awk '{if(NR>1) print $1}')
+    for POD in $PODS
+    do
+      V ${SUDO} crictl rmp $POD 
+    done
+  fi 
+}
```
Comment (copejon) [inline hack/cleanup.sh:59]: I believe these functions could be consolidated into a single line, killing and removing all containers:

`$ until crictl rmp --force --all; do sleep 1; done`

**Review by jbpratt — COMMENTED** [review]
Should the microshift binary also be removed from the system? Cleaning up firewall rules, selinux config, stopping crio and uninstalling packages (if the desire is to have this script clean up everything from the install script) may be needed as well.

**Comment by cooktheryan** [comment]
@guymguym I would say with RPM we would have a more formal removal process. This is in my opinion a nice supplement to help when running the install.sh or doing local development to set you back at a close to beginning state

**Comment by guymguym** [comment]
> @guymguym I would say with RPM we would have a more formal removal process. This is in my opinion a nice supplement to help when running the install.sh or doing local development to set you back at a close to beginning state

👍  Sounds good @cooktheryan !

**Comment by cooktheryan** [comment]
closed will merge items in at #405

**Pull request closed without merging**